meta/recipes-core/meta/meta-toolchain.bb

```diff
@@ -3,4 +3,13 @@ LICENSE = "MIT"
 
 PR = "r7"
 
+python() {
+    # It does not make sense to build a toolchain when BUILD_IMAGES_FROM_FEEDS
```

Comment:
I had difficulty writing this, because I kept wondering if there might be someone out there with a pre-existing feed of the SDK packages for a given SDK_MACHINE, just trying to put together a toolchain. Seems far-fetched, but probable?

Reply:
I can't imagine what use-case that would serve. So I would be comfortable committing this code and letting that theoretical person raise their concern if/when this breaks them. At the least, it will break them verbosely and the workaround is plainly obvious.